aquality-selenium-template/src/main/java/aquality/selenium/template/forms/pages/MainPage.java

```diff
@@ -6,6 +6,6 @@
 public class MainPage extends BaseAppForm {
 
     public MainPage() {
-        super(By.xpath("//section[@class='testing-services-block']"), "Main page");
+        super(By.xpath("//section[@class='services']"), "Main page");
```

Comment:
Should we use ```contains(@class,'')``` for any class in locators, even if it's one? We have a ```<section>``` tags with more than one class.

Reply:
I believe yes, we should

Reply:
fixed